src/bctklib/persistence/StateServiceStore.cs

```diff
@@ -195,8 +196,16 @@ static async Task<IReadOnlyList<ContractInfo>> GetContracts(RpcClient rpcClient,
                         var (key, value) = found.Results[i];
                         if (key.AsSpan().StartsWith(prefix.Span))
                         {
-                            var state = new StorageItem(value).GetInteroperable<ContractState>();
-                            contracts.Add(new ContractInfo(state.Id, state.Hash, state.Manifest.Name));
+                            // Temporary fix --> https://github.com/neo-project/neo/issues/2829
```

Comment:
@merl111 have you actually encountered this problem in the wild with your contracts?

Reply:
Yes,  when trying to sync states after the contract was deployed it fails due to that, also commented  [here](https://github.com/neo-project/neo/issues/2829#issuecomment-1326790496 ).

A possible workaround is to sync states from a block pre contract deployment.